.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: django-allow-cidr PyPI release
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+jobs:
+  pre_release_checks:
+    runs-on: ubuntu-latest
+    name: django-allow-cidr Pre-Release CI checks
+    steps:
+      - name: Run CI checks again before publishing package
+        uses: ./.github/workflows/ci
+
+  release:
+    runs-on: ubuntu-latest
+    name: django-allow-cidr Release
+    needs:
+      - pre_release_checks
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+      - name: Install dependencies for package building only
+        run: pip install build
+      - name: Build package for upload to PyPI
+        run: >-
+          python -m
+          build
+          --sdist
+          --outdir dist/
+          .
+      - name: Upload the distribution to PyPI
+        uses: pypa/gh-action-pypi-publish@v1.5.0
```

Comment:
This action isn't in our allow list for the org. We could either just manually install and use `twine` here or request that this action be added to our allow list for the org.

Reply:
It's already in use for `mozilla`, so I'll ask for it to be added to `mozmeao`

Reply:
Requested in https://bugzilla.mozilla.org/show_bug.cgi?id=1778203

Reply:
@pmac This action is now allowed on mozmeao